wizard/WizardCreateWallet2.qml

```diff
@@ -49,39 +100,353 @@ Rectangle {
+                        tooltipIconVisible: true
+                        themeTransition: false
+                        Accessible.role: Accessible.StaticText
+                        Accessible.name: qsTr("Wallet restore height") + " " + Utils.roundDownToNearestThousand(wizardController.m_wallet ? wizardController.m_wallet.walletCreationHeight : 0) + translationManager.emptyString
+                        KeyNavigation.up: walletCreationDate
+                        KeyNavigation.backtab: walletCreationDate
+                        Keys.onDownPressed: navigation.btnPrev.forceActiveFocus();
+                        Keys.onTabPressed: navigation.btnPrev.forceActiveFocus();
+                    }
+
+                    MoneroComponents.TextPlain {
+                        id: walletRestoreHeightValue
+                        font.family: MoneroComponents.Style.fontRegular.name
+                        font.pixelSize: 16
+                        font.bold: true
+                        textFormat: Text.RichText
+                        color: MoneroComponents.Style.defaultFontColor
+                        text: Utils.roundDownToNearestThousand(wizardController.m_wallet ? wizardController.m_wallet.walletCreationHeight : 0)
+                    }
+                }
             }
 
             WizardNav {
-                progressSteps: appWindow.walletMode <= 1 ? 3 : 4
+                id: navigation
+                progressSteps: appWindow.walletMode <= 1 ? 4 : 5
                 progress: 1
-                btnNext.enabled: passwordFields.calcStrengthAndVerify();
                 onPrevClicked: {
-                    if(wizardController.walletOptionsIsRecoveringFromDevice){
-                        wizardStateView.state = "wizardCreateDevice1";
-                    } else {
-                        wizardStateView.state = "wizardCreateWallet1";
-                    }
+                    wizardStateView.state = "wizardCreateWallet1";
+                    mobileDialog.visible = Qt.binding(function() { return wizardController.layoutScale == 4 })
                 }
+                btnPrevKeyNavigationBackTab: wizardCreateWallet2.state == "default" ? walletRestoreHeight
+                                                                                    : seedListGridColumn.children[0].children[wizardCreateWallet2.hiddenWords[4]].lineEdit.visible ? seedListGridColumn.children[0].children[wizardCreateWallet2.hiddenWords[4]].lineEdit
+                                                                                                                                                                                   : seedListGridColumn.children[0].children[24]
+                btnNextKeyNavigationTab: mobileDialog.visible ? mobileHeader : header
+                btnNext.enabled: walletCreationDate.opacity == 1 || appWindow.ctrlPressed ? true
```

Comment:
Would use here `wizardCreateWallet2.state == "default"` instead of `walletCreationDate.opacity == 1`

I assume they should be equivalent.

Reply:
Doesn't work.